crates/rari-doc/src/reader.rs

```diff
@@ -74,6 +80,7 @@ pub fn read_docs_parallel<P: 'static + Send, T: PageReader<P>>(
                             //tx.send(Err(e.into())).unwrap();
                         }
                     }
+                    drop(enter);
```

Comment:
`enter` is dropped here anyway, since it is the end of its defining block. This can be removed and the `enter` variable renamed to `_enter` or `_`.

Reply:
Good point.

Reply:
```suggestion
```